packages/subapp-util/lib/index.js

```diff
@@ -242,7 +261,12 @@ function loadSubAppByName(name) {
   const container = getSubAppContainer();
   const subAppDir = manifest.subAppDir;
   // load subapp's entry
-  xrequire(Path.resolve(appSrcDir(), subAppDir, manifest.entry));
+  const modSrcDir = manifest.srcDir || "lib";
+  const fullSubappDir = manifest.module
+    ? Path.resolve("node_modules", subAppDir, modSrcDir)
+    : Path.resolve(appSrcDir(), subAppDir);
```

Comment:
seems duplicated below, create a function for this?  especially the srcDir || "lib" logic